<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Customer Relationship Form</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="entryUserName">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>User Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Email</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="entryUserEmail">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>100</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="OutputDisplay">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>210</y>
      <width>571</width>
      <height>261</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnReset">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>40</y>
      <width>92</width>
      <height>36</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Reset input fields to default values</string>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClose">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>90</y>
      <width>92</width>
      <height>36</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Close this application</string>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSubmit">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>140</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Create a desktop shortcut for this applicatio</string>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
